src/rewrites.lisp

```diff
@@ -44,9 +44,9 @@ Example patterns:
                 (assert (symbolp (car tree)) () "Functions must be constants in current impl")
                 `(application ,(car tree) ,@(mapcar #'compile-tree (cdr tree))))
                (t ; equivalent to (atom tree)
-                (when (and (symbolp tree)
-                           (string= "_" tree))
-                  (warn "Tried to compile a pattern involving a symbol named \"_\" in a package other than CLUCK. This will not be treated as a wildcard!"))
+                (and (symbolp tree)
```

Comment:
keep this a when since warn is effectively side effectful

Reply:
The `warn` is now inside the `and`, the code works the same

Unless you're giving me a style warning 😉

Reply:
yes it's a style warning. AND is for computing values, not having a shortcut for WHEN :)

Reply:
> AND is for computing values, not having a shortcut for WHEN

*cries in `sh`*

Reply:
*crosses pointer fingers perpendicularly such that they lie in a plane whose normal is directed toward `sh`*